package.json

```diff
@@ -7,5 +7,8 @@
     "express": "4.17.1",
     "express-handlebars": "3.1.0",
     "nodemon": "^2.0.15"
+  },
+  "scripts": {
+    "start": "node_modules/.bin/nodemon server.js"
```

Comment:
W package.json już nie musisz podawać ściezki zawierającej `node_moules/.bin/` jesli pakiet jest dodany jako zaleznośc. Jej tylko potrzebujesz aby odpalić pakiet z konsoli. Jeśli użyjesz do tego npm to on sam to ogarnie :)